cmd/chloggen/main.go

```diff
@@ -0,0 +1,228 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	breaking     = "breaking"
+	deprecation  = "deprecation"
+	newComponent = "new_component"
+	enhancement  = "enhancement"
+	bugFix       = "bug_fix"
+
+	insertPoint = "<!-- next version -->"
+)
+
+func main() {
```

Comment:
We use `github.com/spf13/cobra` in this project already. It should simplify all the command args parsing. WDYT?

Reply:
I figured this was simple enough not to bother yet. I don't feel strongly either way, but slightly prefer not to spend the time switching it over. Perhaps a p3 followup issue is appropriate?

Reply:
Yeah p3 issue sounds. Just to reduce amount of code to read :)

Reply:
Done, #11822